roles/setup_lvm_lv/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+debug: False
+
+metal_k8s_lvm_lvs:
+  kubevg:
```

Comment:
We should review this list.

Reply:
Elasticsearch `data`: 50Gi x 3
Elasticsearch `master`: 4Gi x 3
Prometheus: 10Gi x 2

So we should provision a 52GB, 5GB and 11GB volume by default, then document how to add more with Zenko-specific sizing (or an example of "Here's how you configure things to get a pre-0.2 layout")